include/tvm/relay/attrs/transform.h

```diff
@@ -210,14 +210,22 @@ struct SplitAttrs : public tvm::AttrsNode<SplitAttrs> {
 
 /*! \brief Attributes for StridedSlice operator */
 struct StridedSliceAttrs : public tvm::AttrsNode<StridedSliceAttrs> {
-  Array<Integer> begin;
-  Array<Integer> end;
-  Array<Integer> strides;
+  Optional<Array<Integer>> begin;
+  Optional<Array<Integer>> end;
+  Optional<Array<Integer>> strides;
+  bool slice_mode;
 
   TVM_DECLARE_ATTRS(StridedSliceAttrs, "relay.attrs.StridedSliceAttrs") {
     TVM_ATTR_FIELD(begin).describe("Indices for begin of slice, begin index is also inclusive");
     TVM_ATTR_FIELD(end).describe("Indices for end of slice, end index is exclusive");
-    TVM_ATTR_FIELD(strides).set_default(Array<Integer>({})).describe("Stride values of the slice");
+    TVM_ATTR_FIELD(strides).describe("Stride values of the slice");
+    TVM_ATTR_FIELD(slice_mode)
```

Comment:
The `slice_mode` name is a bit confusing.

Reply:
Will ```use_size`` be more clear?

Reply:
how about `as_slice`?

Reply:
We can still call it `slice_mode`, but change it from bool to string option selected from ["end", "size"]. How about this?

Reply:
This sounds great to me.

Reply:
> We can still call it `slice_mode`, but change it from bool to string option selected from ["end", "size"]. How about this?

sounds good to me

Reply:
> > We can still call it `slice_mode`, but change it from bool to string option selected from ["end", "size"]. How about this?
> 
> sounds good to me

Changed.